virttest/utils_test/libvirt.py

```diff
@@ -1592,6 +1592,22 @@ def alter_boot_order(vm_name, pci_id, boot_order=0):
     vmxml.sync()
 
 
+def wait_for_file_over(patt, xml_file, timeout=10):
```

Comment:
I have no problem with the code changes, I just wonder if we need to move it to utils_misc...I think this function can be used widely if do so. How do you think?

Reply:
As most of the usage is for xml writing finish judgement and most of the usage should be in tp-libvirt and in the same file libvirt.py, then I think put it here should be OK.

Reply:
Ok.